### PR TITLE
fix: move to new nginx charts

### PR DESCRIPTION
**Pull request by ankitm123**
Signed-off-by: ankitm123 <ankitmohapatra123@gmail.com>

### CODE REVIEW

File: charts/ingress-nginx/values.yaml.gotmpl [hunk 1]
```diff
@@ -1,20 +1,18 @@
 controller:
   replicaCount: 3
   extraArgs:
-    publish-service: nginx/nginx-ingress-controller
+    publish-service: nginx/ingress-nginx-controller
   service:
-    omitClusterIP: true
```
Comment (ankitm123) [inline charts/ingress-nginx/values.yaml.gotmpl:6]: afaict, this option is deprecated

**Comment by jstrachan** [comment]
@ankitm123 I merged this change into other PRs upgrading jx-gitops and other things in the version stream; all good now thanks!

**Pull request closed without merging**